lib/std/json.zig

```diff
@@ -1688,3 +2244,269 @@ test "string copy option" {
     }
     testing.expect(found_nocopy);
 }
+
+pub const JsonDumpOptions = struct {
+    // TODO: indentation options?
+    // TODO: make escaping '/' in strings optional?
+    // TODO: allow picking if []u8 is string or array?
+};
+
+pub fn dump(
```

Comment:
In the standard library, I've been using `dump` by convention to mean "this function is for debugging purposes and will print to stderr some kind of non-stable representation of the object". Maybe this can be called `serialize`?

Reply:
`serialize` would pair with `deserialize` and I like `parse` better :P

What about `stringify`? (which is [what JS uses](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify))

Reply:
:+1: